apps/voting-disputable/test/contracts/voting_delegation.js

```diff
@@ -27,6 +27,8 @@ contract('Voting delegation', ([_, owner, voter, anotherVoter, thirdVoter, repre
     voting = await deployer.deployAndInitialize({ owner, minimumAcceptanceQuorum: MIN_QUORUM, requiredSupport: MIN_SUPPORT, voteDuration: VOTE_DURATION, overruleWindow: OVERRULE_WINDOW, quietEndingPeriod: QUIET_ENDING_PERIOD, quietEndingExtension: QUIET_ENDING_EXTENSION })
   })
 
+  const repeat = (x, y) => [...Array(x)].map(() => y)
```

Comment:
Note that you can replace this with just `new Array(x).fill(y)` now :)

Reply:
Addressed in beacbd92213037ea26201ea834229ed5f960432b